### PR TITLE
chore(1.9.0): Update litmus helm to 1.9.0 

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

#### What this PR does / why we need it:

- This PR is for adding the flag with the experiment binary. This is according to the latest change in the experiment CR now the `./experiment` contains all experiments in this binary So to run the selective experiment we need to pass a flag accordingly. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] DCO signed
- [X] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Review by ispeakc0de — APPROVED** [review]

**Comment by ksatchit** [comment]
@uditgaurav lets convert this into the release update PR for litmus & Kubernetes-chaos helm chart

File: charts/kubernetes-chaos/templates/container-kill.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
     imagePullPolicy: {{ .Values.image.litmusGO.pullPolicy }}
     args:
     - -c
-    - ./experiments/container-kill
+    - ./experiments -name container-kill
```
Comment (ispeakc0de) [inline charts/kubernetes-chaos/templates/container-kill.yaml:44]: sequence env?

File: charts/kubernetes-chaos/templates/pod-cpu-hog.yaml [hunk 2]
```diff
@@ -64,6 +64,11 @@ spec:
     - name: LIB
       value: 'litmus'
 
+    ## it defines the sequence of chaos execution for multiple target pods
+    ## supported values: serial, parallel
+    - name: SEQUENCE
```
Comment (ispeakc0de) [inline charts/kubernetes-chaos/templates/pod-cpu-hog.yaml:69]: would you like to add `chaos_kill/inject_command` in both cpu/memory hog experiment

File: charts/litmus/Chart.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "1.8.2"
+appVersion: "1.9.0"
 description: A Helm chart to install litmus infra components on Kubernetes
 name: litmus
-version: 1.8.2
+version: 1.9.2
```
Comment (ispeakc0de) [inline charts/litmus/Chart.yaml:5]: 1.9.0?

**Comment by ksatchit** [comment]
Updated permissions in operator & admin-mode rbac needs to be added!

**Review by ksatchit — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Comment by calvinbui** [comment]
looking at the version number, it should've been `2.1.3` -> `2.2.0`, not `2.2.3`

by changing it to `2.2.3` it suggests there were 3 bugfixes (`2.2.1`, `2.2.2`, `2.2.3`) since `2.2.0`.

too late to change it now, but a note for next time

**Pull request merged**